pyqt_sql_demo/connection/model.py

```diff
@@ -2,17 +2,22 @@
 from PyQt5.QtCore import pyqtSignal
 
 import pyqt_sql_demo.connection.exceptions as exceptions
+import pyqt_sql_demo.widgets.text as UI
+
+from PyQt5.QtWidgets import QApplication, QInputDialog, QLineEdit
 
 import os.path
 import sqlite3
+import mysql.connector as MySqlConnector
 import logging
-
+import typing # Needed for byte string to string conversion
 
 class ConnectionModel(QAbstractTableModel):
     executed = pyqtSignal(str, name="executed")
     connected = pyqtSignal(str, name="connected")
     disconnected = pyqtSignal()
     fetch_changed = pyqtSignal(bool, name="fetch_changed")
+    database_type = None
```

Comment:
`database_type` here is a class instance level variable. The database type value should not be shared between all instances of the `ConnectionModel` class in case we ever instantiate more than one. Instead, it should be declared inside with __init__ method.